pom.xml

```diff
@@ -61,7 +61,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-shared-dependencies</artifactId>
-        <version>2.3.0</version>
+        <version>2.4.0</version>
```

Comment:
Before declaring google-auth-library-oauth2-http, this failed: `package com.google.auth.oauth2 does not exist` https://github.com/googleapis/java-pubsub/pull/882#issuecomment-946272895